easybuild/easyconfigs/c/ConnectomeWorkbench/ConnectomeWorkbench-1.3.2.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+# Version 1.3.2 modified by:
+# Adam Huffman
+# adam.huffman@bdi.ox.ac.uk
+
+easyblock = "Tarball"
+
+name = 'ConnectomeWorkbench'
+version = '1.3.2'
+
+homepage = 'http://www.humanconnectome.org/software/connectome-workbench.html'
+description = """Connectome Workbench is an open-source 
+visualization and discovery tool used to explore data generated 
+by the Human Connectome Project. The distribution includes wb_view, 
+a GUI-based visualization platform, and wb_command, a command-line 
+program for performing a variety of algorithmic tasks using volume, 
+surface, and grayordinate data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://ftp.humanconnectome.org/workbench/']
+sources = ['workbench-rh_linux64-v%(version)s.zip']
```

Comment:
@verdurin Doesn't this deserve a `-rh` `versionsuffix`, to discriminate between the CentOS/RedHat and Ubuntu/Debian builds that are provided? Cfr. https://www.humanconnectome.org/software/get-connectome-workbench